src/interfaces/redux.ts

```diff
@@ -53,40 +54,20 @@ export interface CanisterInfo {
   symbol?: string;
 }
 
-interface Currency {
-  symbol: string;
-  decimals: number;
-}
-
-export interface TransactionDetails {
-  status: string; //check if this is correct
-  fee: {
-    amount: string;
-    currency: Currency;
-  };
-  from: string;
-  amount: string;
-  currency: Currency;
-  to: string;
-  caller: string;
-}
-
 export interface Transaction {
-  amount: string | number;
-  type: string; //TODO: Add types here SEND/RECEIVE. Check ACTIVITY_TYPES
-  symbol: string;
-  hash: string;
+  type: string;
   to: string;
   from: string;
-  date: Date;
-  image: string;
-  value?: string | number;
-  status?: number | string;
-  icon?: string;
-  canisterId?: string;
-  plug?: any;
-  canisterInfo?: CanisterInfo;
-  details?: TransactionDetails;
+  hash: string;
+  amount: Nullable<number | typeof NaN>;
+  value?: Nullable<number>;
+  status: number;
+  date: bigint;
+  symbol: string;
+  logo: string;
+  canisterId: string;
+  details?: { [key: string]: any };
+  canisterInfo?: Object;
```

Comment:
what about the interface `CanisterInfo` that we have above?

Reply:
I use the same one as in the Controller cuz otherwise i was getting problems with TS